libpod/options.go

```diff
@@ -436,6 +437,22 @@ func WithRenumber() RuntimeOption {
 	}
 }
 
+// WithMigrate instructs libpod to perform a lock migrateing while
+// initializing. This will handle migrations from early versions of libpod with
```

Comment:
Isn't this not lock-related so much as pidfile-related? I'd call Migrate a generic way to update container configuration between versions